package.json

```diff
@@ -27,7 +27,7 @@
   "homepage": "https://github.com/uc-cdis/guppy#readme",
   "dependencies": {
     "@elastic/elasticsearch": "^7.0.0-rc.1",
-    "@gen3/ui-component": "0.3.10",
+    "@gen3/ui-component": "github:uc-cdis/gen3-ui-component.git#feat/tooltip-filter",
```

Comment:
Just confirming -- we'll pin this to the new version of `gen3-ui-component` once it's published, right?

Reply:
definitely 😺